### PR TITLE
Minor forensic additions

### PR DESCRIPTION
**Pull request by Casper3667**
- Adds helpful descriptions to various forensic tools on how to use them.
- The slides now put items into a free hand if possible when emptying them.
- The fingerprint powder & fiber kit will now only be put into bags or the like if on help intent.
- Swabs will now take GSR from gloves if a person wears them, and won't be used if the person has no GSR on them.

### CODE REVIEW

**Comment by Casper3667** [comment]
!review

**Review by Karolis2011 — APPROVED** [review]

File: code/modules/detectivework/microscope/microscope.dm [hunk 1]
```diff
@@ -2,6 +2,8 @@
 /obj/machinery/microscope
 	name = "high powered electron microscope"
 	desc = "A highly advanced microscope capable of zooming up to 3000x."
+	desc_info = "Use a microscope slide or a fingerprint card on this machine to insert it.\
+	\nAlt click to remove any object within it."
```
Comment (alsoandanswer) [inline code/modules/detectivework/microscope/microscope.dm:6]: This looks a little funny. This is just a linebreak right?

File: code/modules/detectivework/tools/evidencebag.dm [hunk 2]
```diff
@@ -3,6 +3,7 @@
 /obj/item/evidencebag
 	name = "evidence bag"
 	desc = "An empty evidence bag."
+	desc_info = "To use: Click drag it onto an object. Click it in-hand to remove an object from it."
```
Comment (alsoandanswer) [inline code/modules/detectivework/tools/evidencebag.dm:6]:
```suggestion
	desc_info = "Click drag this onto an object to put it inside. Click it in-hand to remove an object from it."
```

File: code/modules/detectivework/microscope/microscope.dm (hunk 1 above)
Comment (Casper3667) [reply code/modules/detectivework/microscope/microscope.dm:6]: Yes. I added them to the descriptions to make it easier to read.

File: code/modules/detectivework/tools/sample_kits.dm [hunk 3]
```diff
@@ -51,11 +51,15 @@
 /obj/item/sample/fibers
 	name = "fiber bag"
 	desc = "Used to hold fiber evidence for the detective."
+	desc_info = "Holds various fibe evidence. Place it in a slide and the slide into a microscope to check them."
```
Comment (Arrow768) [inline code/modules/detectivework/tools/sample_kits.dm:54]:
```suggestion
	desc_info = "Holds various fibre evidence. Place it in a slide and the slide into a microscope to check them."
```

**Review by Arrow768 — APPROVED** [review]

**Review by alsoandanswer — APPROVED** [review]

**Pull request merged**